src/internal/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { intersection, mergeWith, omit } from 'lodash';
+import mergeWith = require('lodash/mergeWith');
```

Comment:
Yes, if we make it like `import mergeWith from 'lodash.mergeWith'` it gives an error in build

Reply:
shouldnt `lodash/mergeWith` just work, or is this just valid if it a path *inside* a package?

Reply:
`lodash/mergeWith` works when `lodash` is installed

I uninstalled whole lodash package and installed only mergeWith library `lodash.mergeWith` to reduce the size

Reply:
then unless this function can also be de-lodash'ed, i dont think i will go to smaller packages, [because of this quote](https://lodash.com/per-method-packages):

> However, use of these packages is discouraged and they [will be removed in v5](https://github.com/lodash/lodash/issues/3838#issuecomment-398592530).